src/installer/pkg/projects/Directory.Build.targets

```diff
@@ -14,6 +14,17 @@
     <StableVersion Condition="'$(IncludePreReleaseLabelInPackageVersion)' != 'true'">$(Version)</StableVersion>
   </PropertyGroup>
 
+  <!--
+    For mobile targets, we do not need the host, so allow depproj's to opt 
+    in in order to skip building. 
+  -->
+  <Target Name="MobileSkipBuildProps" BeforeTargets="GetSkipBuildProps" 
+    Condition="'$(SkipBuildOnRuntimePackOnlyOS)' and '$(TargetOS)' == 'iOS'">
```

Comment:
(Nit) I find this style of wrapping significantly harder to read quickly vs. chopping or alignment. `BeforeTargets` is hard to see, and `Condition` being left of `Name` is unusual. This is a thing that maybe you can get used to, but thought I'd mention it anyway while I'm here and because it's in src/installer.